src/core/stdc/math.d

```diff
@@ -3500,81 +3500,81 @@ else version(DragonFlyBSD)
     float ynf(int, float);
     float gammaf(float x);
     float dremf(float x, float);
-    int finitef(float x);
-    int isinff(float x);
-    int isnanf(float x);
+    pure int finitef(float x);
+    pure int isinff(float x);
+    pure int isnanf(float x);
 
     float gammaf_r(float x, int *);
     float lgammaf_r(float x, int *);
     float significandf(float x);
 
     /* real */
-    real acosl(real x);
-    real asinl(real x);
-    real atanl(real x);
-    real atan2l(real x, real y);
-    real cosl(real x);
-    real sinl(real x);
-    real tanl(real x);
-
-    real acoshl(real x);
-    real asinhl(real x);
-    real atanhl(real x);
-    real coshl(real x);
-    real sinhl(real x);
-    real tanhl(real x);
-
-    real expl(real x);
-    real exp2l(real x);
-    real expm1l(real x);
-    real frexpl(real x, int *exp);
+    pure real acosl(real x);
+    pure real asinl(real x);
+    pure real atanl(real x);
+    pure real atan2l(real x, real y);
+    pure real cosl(real x);
+    pure real sinl(real x);
+    pure real tanl(real x);
+
+    pure real acoshl(real x);
+    pure real asinhl(real x);
+    pure real atanhl(real x);
+    pure real coshl(real x);
+    pure real sinhl(real x);
+    pure real tanhl(real x);
+
+    pure real expl(real x);
+    pure real exp2l(real x);
+    pure real expm1l(real x);
+    pure real frexpl(real x, int *exp);
     int ilogbl(real x);
-    real ldexpl(real x, int exp);
+    pure real ldexpl(real x, int exp);
     real logl(real x);
     real log10l(real x);
     real log1pl(real x);
     real log2l(real x);
-    real logbl(real x);
-    real modfl(real x, real *iptr);
-    real scalbnl(real x, int y);
-    real scalblnl(real x, c_long y);
-
-    real cbrtl(real x);
-    real fabsl(real x);
-    real hypotl(real x, real y);
-    real powl(real x, real y);
-    real sqrtl(real x);
-
-    real erfl(real x);
-    real erfcl(real x);
-    real lgammal(real x);
-    real tgammal(real x);
-
-    real ceill(real x);
-    real floorl(real x);
-    real nearbyintl(real x);
-    real rintl(real x);
+    pure real logbl(real x);
+    pure real modfl(real x, real *iptr);
+    pure real scalbnl(real x, int y);
+    pure real scalblnl(real x, c_long y);
+
+    pure real cbrtl(real x);
+    pure real fabsl(real x);
+    pure real hypotl(real x, real y);
+    pure real powl(real x, real y);
+    pure real sqrtl(real x);
```

Comment:
For non-DragonFlyBSD `cbrt` and `fabs` are marked pure but not `hypot`, `pow`, `sqrt`.